app/views/collections/show.rb

```diff
@@ -30,10 +30,8 @@ def head_meta
       end
     end
 
-    def page_title
-      mustache[:page_title] ||= begin
-        @landing_page.title || @collection.title
-      end
+    def page_content_heading
+      @collection.title
```

Comment:
we should probably still use the landing_page title as a preference since this gives editors control over the title without changing the collection name, which only administrators can do.
also by removing the mutache[:page_title] this is being called twice now. you could assign it into mustache[:page_content_heading]